autocomplete/definitions/namedTypes/tes3vector2/distanceChebyshev.lua

```diff
@@ -0,0 +1,17 @@
+return {
+	type = "method",
+	description = [[Calculates the distance to another vector, using the Chebyshev metric, which is defined as
+
+	math.max(math.abs(v1.x - v2.x), math.abs(v1.y - v2.y))
+
+This is useful for ensuring that the x and y coordinates between two vectors are all (independently) within a certain distance from each other.
```

Comment:
If you want, you can also link the Wiki article here. The comparison between the three distances here is nice:
https://en.wikipedia.org/wiki/Euclidean_distance#Generalizations
or link the article on Chebyshev's distance.

But this isn't necessary. I really appreciate your geometric description. You taught me something new today.